bindings/python/tools/python/ostk/astrodynamics/czml.py

```
@@ -0,0 +1,330 @@
+        instants (list[Instant]): The sample instants.
+        quaternions (list): The sample quaternions, as a list of [x, y, z, w] sequences,
+            mapping the entity body frame into the Earth-fixed frame.
+
+    Returns:
+        dict: The CZML sampled property.
+    """
+
+    epoch: Instant = instants[0]
+
+    samples: list[float] = []
+    for instant, quaternion in zip(instants, quaternions):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)